tf2onnx/tfonnx.py

```diff
@@ -1747,6 +1751,56 @@ def sparse_softmax_cross_entropy_with_logits_op(ctx, node, name, args):
     return [onehot, log_softmax, mul1, reduce_sum, mul2, res]
 
 
+def floordiv_op(ctx, node, name, args):
+    # T output = floordiv(T x, T y)
+    res = [node]
+    node.type = "Div"
+    dtype = ctx.get_dtype(node.input[0])
+    if dtype in [onnx_pb.TensorProto.FLOAT, onnx_pb.TensorProto.FLOAT16, onnx_pb.TensorProto.DOUBLE]:
+        new_node_name = utils.make_name("floor_div_res")
+        floor_res = ctx.insert_new_node_on_output(op_type="Floor", output_name=node.output[0],
+                                                  name=new_node_name)
+        ctx.copy_dtype(node.output[0], floor_res.output[0])
+        ctx.copy_shape(node.output[0], floor_res.output[0])
+        res.append(floor_res)
+    return res
+
+
+def floormod_op(ctx, node, name, args):
+    # T output = floormod(T x, T y)
+    nodes = []
+    div = ctx.make_node(op_type="Div", inputs=node.input)
+    nodes.append(div)
+    dtype = ctx.get_dtype(node.input[0])
+    if dtype in [onnx_pb.TensorProto.FLOAT, onnx_pb.TensorProto.FLOAT16, onnx_pb.TensorProto.DOUBLE]:
+        div = ctx.make_node(op_type="Floor", inputs=div.output)
+        nodes.append(div)
+    mul = ctx.make_node(op_type="Mul", inputs=[div.output[0], node.input[1]])
+    # res node will take over shape&dtype&output connection info of original "node"
+    res = ctx.make_node(op_type="Sub", inputs=[node.input[0], mul.output[0]],
+                        name=node.name, outputs=node.output)
+    nodes.extend([mul, res])
+    return nodes
+
+
+def all_op(ctx, node, name, args):
+    # T output = reduce_all(T x, list(int) reduce_indices, @bool keepdims)
+    reduce_dim = node.inputs[1].get_tensor_value()
+    if isinstance(reduce_dim, np.ndarray):
+        reduce_dim = reduce_dim.tolist()
+    else:
+        reduce_dim = [reduce_dim[0]]
+    for i in reduce_dim:
+        utils.make_sure(i >= 0, "negative reduce axis is not supported in onnx for now")
+
+    cast = ctx.make_node(op_type="Cast", inputs=[node.input[0]], attr={"to": onnx_pb.TensorProto.INT32})
```

Comment:
you don't need specify op_type each time actually.

Reply:
i thought this could increase the readability of code.